### PR TITLE
Upgrade from .net core RC1 (dnvm/dnu/dnx) to RC2 (dotnet)

### PR DESCRIPTION
**Pull request by qooroo**
Do not merge yet - need to test with circle first

### CODE REVIEW

File: deploy/docker/build/servers/template.Dockerfile [hunk 1]
```diff
@@ -3,7 +3,8 @@ MAINTAINER  weareadaptive <thibault@weareadaptive.com>
 
 COPY        server    /server
 
-ENV         PATH    /root/.dnx/runtimes/dnx-mono.__VDNX__/bin:$PATH
+# TODO: check that we don't need this
+# ENV         PATH    /root/.dnx/runtimes/dnx-mono.__VDNX__/bin:$PATH
```
Comment (tdeheurles) [inline deploy/docker/build/servers/template.Dockerfile]: if the server run with success you don't need.

**Comment by tdeheurles** [comment]
fine for the devops part

File: deploy/docker/build/servers/template.Dockerfile (hunk 1 above)
Comment (qooroo) [inline deploy/docker/build/servers/template.Dockerfile]: yup I've just tested it - I'll take it out

**Comment by qooroo** [comment]
superceded by #425

**Pull request closed without merging**